ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-sdk-e2e-tests.robot

```diff
@@ -91,6 +99,14 @@ Run Codeflare-sdk E2E Test
     [Arguments]    ${TEST_NAME}
     Log To Console    "Running codeflare-sdk test: ${TEST_NAME}"
     ${result} =    Run Process  source ${VIRTUAL_ENV_NAME}/bin/activate && cd codeflare-sdk && poetry env use 3.9 && poetry install --with test,docs && poetry run pytest -v -s ./tests/e2e/${TEST_NAME} --timeout\=600 && deactivate
+    ...    env:RAY_IMAGE=${RAY_IMAGE}
```

Comment:
Why do you need to pass the RHOAI ray image as enviroment variable ?

Reply:
Actually this image will be a common image for both connected and disconnected cluster.
Thanks @ChughShilpa for reviewing !

Reply:
quay.io/rhoai/ray:2.23.0-py39-cu121 now used as default for raycluster-configuration n codeflare-sdk.
 Then why we need to specify it explicitly ?

Reply:
Apologies for any confusion caused, for the `RAY_IMAGE` env variable we will pass a ray image with digest instead of image tag which will be supported by both connected and disconnected cluster
Similar to : 
> RAY_IMAGE=quay.io/rhoai/ray@sha256:859f5c41d41bad1935bce455ad3732dff9d4d4c342b7155a7cd23809e85698ab

Reply:
Make sense now. 
Tags are not supported in disconnected cluster